src/pwm.rs

```diff
@@ -209,6 +234,93 @@ pub struct C2;
 pub struct C3;
 pub struct C4;
 
+/*
+  The following implemention of the embedded_hal::Pwm uses Hertz as a time type.  This was choosen 
+  because of the timescales of operationsbeing on the order of nanoseconds and not being able to 
```

Comment:
operations being

Reply:
"Operations" in this context was intended to be the bus clock period.  Maybe the text here needs to be cleaned up a bit.  I was trying to help anybody who was reading this understand why the units of my time type is inverse time.

Reply:
Oh sorry, I guess I wasn't clear here. It currently says "operationsbeing", but should be "operations being" with a space :)

Reply:
Corrected